transports/msmq/index.md

```diff
@@ -77,25 +77,24 @@ Although MSMQ has the concept of both [Public and Private queues](https://techne
 |----------------|---------------------|-----------------------------------|
 | Owning account | Send, Receive, Peek | Set by NServiceBus                |
 | Administrators | Full                | Set by NServiceBus                |
-| Anonymous      | Send                | Set by NServiceBus prior to 6.1.0 |
-| Everyone       | Send                | Set by NServiceBus prior to 6.1.0 |
+| Anonymous      | Send                | Set by NServiceBus in Versions 6.0.x and below |
+| Everyone       | Send                | Set by NServiceBus in Versions 6.0.x and below |
 
+NOTE: Starting with Versions 6.1.0 and above, the installers will not automatically grant permissions to the `Anonymous` and `Everyone` group. It will respect the existing queue permissions that have been set up for the endpoint. 
```

Comment:
"Starting with" does not flow well with "Versions X and above". What about just "In Version 6.1.0 and above"?

There's two key differences in that version, isn't there? Firstly, we no longer change permissions on existing queues at all (in the past we did). Secondly, we no longer set permissions for Anonymous and Everyone explicitly. What happens with those accounts is a windows default for all newly created queues (created by NSB or any other mechanism). I wonder if we need to be more explicit about that.